### DISCUSSION TITLE
No "Your organization"

### DISCUSSION THREAD

**Discussion by Veguk03** [community/community#56451] [New to GitHub]
Hello help please.When i click account menu, i didnt have "Your organiztion".How i can fixed?

**Comment by queenofcorgis** [comment]
Hi there - are you a part of any organizations?

**Comment by Veguk03** [comment]
Yes i am part of organization "skill factory"

вт, 30 мая 2023 г., 11:43 PM Lili ***@***.***>:

**Reply by priyankarpal** [reply]
you need to make yourself public in a org after the org will show on your github account

this is the demo 

![image](https://github.com/community/community/assets/88102392/21a2d2cc-c05e-430f-bbcf-b449e4cc9bcf)

### How to make public your account
- just go to the org > select people > search your username > & change to `public`

![image](https://github.com/community/community/assets/88102392/c5b61266-5dd4-4bad-9740-54dcd61fd08c)
